.azp-private.yml

```diff
@@ -22,6 +22,7 @@ resources:
     type: github
     endpoint: cocotb
     name: cocotb/cocotb-private-ci
+    ref: refs/heads/stable/1.4
```

Comment:
What's the purpose of this?

Reply:
To use the CI job description corresponding to the stable branch, reducing potential breakage as we change private CI to match things on master. I didn't create the branch, but I like the idea.

Reply:
Ah, so this refers to a branch within `cocotb/cocotb-private-ci`. I'm surprised this works if the branch doesn't exist.

Reply:
Yeah I created the branch and made this commit after I was seeing CI not passing because of the VHDL tests which were added after the release.